docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -74,10 +74,10 @@ <h5>
             <th>Resolution</th>
           </tr>
           <tr>
-            <td>999</td>
-            <td>Put first resolved issue in 5.3.1 title here</td>
+            <td>1339</td>
+            <td><code>run_historical.sh</code> script results in Blocked databases</td>
```

Comment:
I think this should be a bit more generic sounding
```suggestion
            <td>Building with incomplete assets can result in Blocked databases</td>
```